WordPress/Classes/ViewRelated/Me/App Settings/AppSettingsViewController.swift

```diff
@@ -475,58 +485,84 @@ private extension AppSettingsViewController {
         )
     }
 
-    func otherTableSection() -> ImmuTableSection {
-        let otherHeader = NSLocalizedString("Other", comment: "Link to About section (contains info about the app)")
+    func appearanceTableSection() -> ImmuTableSection {
+        var rows: [ImmuTableRow] = []
 
-        let debugRow = NavigationItemRow(
-            title: NSLocalizedString("Debug", comment: "Navigates to debug menu only available in development builds"),
-            icon: .gridicon(.bug),
-            action: pushDebugMenu()
+        let colorModeRow = NavigationItemRow(
+            title: NSLocalizedString("settings.color.mode.title", comment: "The title for the app 'Color Mode' setting."),
```

Comment:
Thanks for adding the keys to the strings! This is pretty close but it's missing the `value` now:
```suggestion
            title: NSLocalizedString("settings.color.mode.title", value: "Color Mode", comment: "The title for the app 'Color Mode' setting."),
```

Same comment applies to the other strings.

Reply:
Oops, sorry about that, it is now [fixed](https://github.com/wordpress-mobile/WordPress-iOS/pull/20313/commits/fe5c3c2d8df764944164cf7a42193457cbf29718).